tutorials/build-golo.md

```diff
@@ -0,0 +1,197 @@
+# Build Golo within a Virtual Machine
+
+Building **Golo** is something easy:
+
+```shell
+git clone https://github.com/eclipse/golo-lang.git
+cd golo-lang
+./gradlew installDist
+```
+
+But sometimes, you don't want to install all the needed tools on your laptop (for good or bad reasons). The goal of this document is to explain how to build **Golo** within a virtual machine.
+
+We'll use **[Multipass](https://multipass.run/)** (from **Canonical**) which exists on the leading platforms (Linux, OSX and Windows). Multipass is a lightweight VM manager, simple and easy to use.
+
+> ✋ **Requirements**:
+> - Install Multipass, see the [documentation](https://multipass.run/docs)
+> - Ensure `JAVA_HOME` environment variable is set and points to your JDK installation
+
+## Create the VM and build Golo distribution
+
+- First, create a working directory.
+- In this directory, create two files: `create-vm.sh` and `cloud-init.yaml`.
+- Give the execution rights to `create-vm.sh`.
+
+```shell
+touch create-vm.sh
+touch cloud-init.yaml
+chmod +x create-vm.sh
+```
+
+Add this content to `cloud-init.yaml`
+
+```yaml
+package_update: true
+```
+
+And this is the content of `create-vm.sh`:
+
+```shell
```

Comment:
I don't know if github markdown flavour can include external files (asciidoc can) but if possible, including the script directly would be less error prone...

Reply:
No you can't in plain Markdown

Reply:
markdown cannot